src/main/java/com/ays/user/controller/UserController.java

```diff
@@ -47,7 +47,7 @@ class UserController {
      * @param listRequest The request object containing the list criteria.
      * @return A response object containing a paginated list of users.
      */
-    @PostMapping
+    @GetMapping
```

Comment:
Can we revert this since it was edited in a different branch?

Reply:
i reverted.